quarkus_quickstarts_test failed with Could not find the selected project in the reactor: :hibernate-search-elasticsearch-quickstart

quarkus_quickstarts_test failed on all platforms.

internal/Test_openjdk11_j9_sanity.external_x86-64_linux/49 output:

```
Downloaded from central: https://repo.maven.apache.org/maven2/org/optaplanner/optaplanner-parent/8.16.1.Final/optaplanner-parent-8.16.1.Final.pom (5.1 kB at 69 kB/s)
00:09:36.475  [ERROR] [ERROR] Could not find the selected project in the reactor: :hibernate-search-elasticsearch-quickstart @ 
00:09:36.475  [ERROR] Could not find the selected project in the reactor: :hibernate-search-elasticsearch-quickstart -> [Help 1]
00:09:36.475  [ERROR] 
00:09:36.475  [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
00:09:36.475  [ERROR] Re-run Maven using the -X switch to enable full debug logging.
00:09:36.475  [ERROR] 
00:09:36.475  [ERROR] For more information about the errors and possible solutions, please read the following articles:
00:09:36.475  [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MavenExecutionException
00:09:36.900  Build quarkus_quickstarts completed
00:09:36.900  Test results copied
00:09:37.303  
00:09:37.303  quarkus_quickstarts_test_0_FAILED
```

The last passed build was Feb 4th Test_openjdk11_j9_sanity.external_x86-64_linux/44 

We can reproduce the error in Grinder:
- openj9: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4147/console
- hotspot: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/4148/console

--- a/external/quarkus_quickstarts/test.properties
+++ b/external/quarkus_quickstarts/test.properties
@@ -1,11 +1,15 @@
 github_url="https://github.com/quarkusio/quarkus-quickstarts.git"
 test_command="mvn -pl !:hibernate-orm-quickstart,!:hibernate-orm-panache-quickstart,\
-!:hibernate-search-elasticsearch-quickstart,!:mqtt-quickstart,\
-!:quartz-quickstart,!:security-jdbc-quickstart,!:security-keycloak-authorization-quickstart,\
+!:hibernate-orm-multi-tenancy-quickstart,\
+!:hibernate-search-orm-elasticsearch-quickstart,\
+!:rabbitmq-quickstart-processor,\
+!:mqtt-quickstart,!:quartz-quickstart,\
+!:redis-quickstart,\
+!:security-jdbc-quickstart,!:security-keycloak-authorization-quickstart,\
 !:security-openid-connect-web-authentication-quickstart,\
 !:security-openid-connect-multi-tenancy-quickstart,!:spring-data-jpa-quickstart,\
-!:vertx-quickstart,!:context-propagation-quickstart,!:getting-started-reactive-rest,\
-!:kafka-quickstart,!:neo4j-quickstart clean install"
+!:vertx-quickstart,!:kafka-quickstart,\
+!:neo4j-quickstart,!:rest-client-quickstart,!:rest-client-multipart-quickstart clean install
 test_results="testResults"
 tag_version="2.7.2.Final"
 environment_variable="MODE=java"

--- a/external/quarkus_quickstarts/test.sh
+++ b/external/quarkus_quickstarts/test.sh
@@ -19,12 +19,16 @@ echo_setup
 export MAVEN_OPTS="-Xmx1g"
 echo "Compile and run quarkus_quickstarts tests"
 mvn --batch-mode -pl !:hibernate-orm-quickstart,!:hibernate-orm-panache-quickstart,\
-!:hibernate-search-elasticsearch-quickstart,!:mqtt-quickstart,\
-!:quartz-quickstart,!:security-jdbc-quickstart,!:security-keycloak-authorization-quickstart,\
+!:hibernate-orm-multi-tenancy-quickstart,\
+!:hibernate-search-orm-elasticsearch-quickstart,\
+!:rabbitmq-quickstart-processor,\
+!:redis-quickstart,\
+!:mqtt-quickstart,!:quartz-quickstart,\
+!:security-jdbc-quickstart,!:security-keycloak-authorization-quickstart,\
 !:security-openid-connect-web-authentication-quickstart,\
 !:security-openid-connect-multi-tenancy-quickstart,!:spring-data-jpa-quickstart,\
-!:vertx-quickstart,!:context-propagation-quickstart,!:getting-started-reactive-rest,\
-!:kafka-quickstart,!:neo4j-quickstart,!:rest-client-quickstart,!:rest-client-multipart-quickstart clean install
+!:vertx-quickstart,!:kafka-quickstart,\
+!:neo4j-quickstart,!:rest-client-quickstart,!:rest-client-multipart-quickstart clean install
 test_exit_code=$?
 echo "Build quarkus_quickstarts completed"
 

